Islandora 1932

JojoVes opened the pull request:
**JIRA Ticket**: [ISLANDORA-1932](https://jira.duraspace.org/browse/ISLANDORA-1932)

* Other Relevant Links [wiki page with correct information](https://wiki.duraspace.org/display/ISLANDORA/Islandora+Batch)

# What does this Pull Request do?

Stresses the importance of using the full path from root for the `--target` or `--scan_target` options, and clarifies clashing information caused by typos regarding `--target` and `--scan_target` for changes between Drush 6 and Drush 7.

# What's new?
Not much new material introduced, just a simple documentation fix to lessen confusion. No new information added, but previously stated information is now made clear.

# How should this be tested?

* Verify that the information matches the wiki.
* Verify that wording is clear and free of typos.

# Additional Notes:
This is a pretty small documentation fix; it shouldn't cause any trouble.

# Interested parties
@Islandora/7-x-1-x-committers

JojoVes committed:
Fixes '--target' and '--scan_target' mix-up

The bigger examples were mixed up even though the correct syntax is specified earlier in the README.

JojoVes committed:
Clarification

`target` requires full path. Clarification taken from the wiki.

willtp87 commented:
The note about absolute path should probably be present on the description of the options in the Drush command.

JojoVes committed:
Further clarification of full path for `target`

Includes the note about absolute path in the description of the options in the Drush command.

JojoVes committed:
Removes unnecessary parentheses

JojoVes committed:
More clarification

Adds clarification in another spot about `target` option needing the full path.
